docs/faq.rst

```diff
@@ -76,6 +76,28 @@ General
 
 General questions regarding library usage belong here.
 
+Are there any usage examples?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Example code can be found in the `examples/ <https://github.com/Rapptz/discord.py/tree/master/examples>`_ folder
+in the repository.
+
+Can I search messages in channels?
```

Comment:
I don't like this question's title nor its content. It needs a rewrite.

Reply:
I ended up deleting that entry because I'm not sure how best to go about it.